src/components/Hero/ContentMedia/index.tsx

```diff
@@ -19,23 +18,34 @@ export const ContentMediaHero: React.FC<
 > = ({ richText, media, mediaWidth, breadcrumbs, links }) => {
   return (
     <Gutter>
-      <Grid className={mediaWidth === 'wide' ? classes.wideGrid : classes.grid}>
-        <Cell start={1} cols={mediaWidth === 'wide' ? 5 : 7} colsM={8}>
+      <div
+        className={[mediaWidth === 'wide' ? classes.wideGrid : classes.grid, 'grid']
```

Comment:
@paulpopus Will need to switch the class name of `classes.grid` to `classes.normalGrid` or similar - that way `classes.grid` is not overlapping with `'grid'`

Does this make sense?